docker-compose.yaml

```diff
@@ -47,7 +47,9 @@ services:
       context: .
       dockerfile: docker/Dockerfile.dev-geth
     restart: on-failure
-    network_mode: host
```

Comment:
`network_mode: host` is not supported on Apple Silicon 😞.

Reply:
How are you able to run tests then?

Reply:
This is the only thing I'm unsure about, this is necessary so that `dev-geth` can expose the ports that `geth` is listening to.